src/main/webapp/scholarx/2020/index.html

```diff
@@ -482,6 +482,24 @@ <h5 class="mb-0">
     </div>
 </section>
 
+<section class="section bg-secondary">
+    <div id="past-onelives" class="container">
+        <div class="row justify-content-center">
+            <div class="col-9 text-center">
+                <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+                <p class="display-3 text-onelive font-weight-bolder">ScholarX</p>
+            </div>
+        </div>
+        </div>
+        <div class="row justify-content-center mb-md-5">
+            <div class="col-9 text-center">
+                <a href="../past-scholarx.html#past-scholarx"
```

Comment:
The correct link should be  `<a href="../archives`

Reply:
I resolved it.